diff --git a/Makefile b/Makefile
index d92736a..63978cd 100644
--- a/Makefile
+++ b/Makefile
@@ -94,8 +94,8 @@ install:
 	install -m 644 -t "$(DESTDIR)/usr/share/binfmts" "$(TOPDIR)/conf/binfmts/dbp.binfmt"
 	install -d "$(DESTDIR)/usr/share/applications"
 	install -m 644 -t "$(DESTDIR)/usr/share/applications" "$(TOPDIR)/conf/dbp-run-path.desktop"
-	install -d "$(DESTDIR)/usr/share/mime"
-	install -m 644 -t "$(DESTDIR)/usr/share/mime" "$(TOPDIR)/conf/mime/x-dbp.xml"
+#	install -d "$(DESTDIR)/usr/share/mime"
+#	install -m 644 -t "$(DESTDIR)/usr/share/mime" "$(TOPDIR)/conf/mime/x-dbp.xml"
 	install -d "$(DESTDIR)/usr/bin"
 	install -m 755 $(STRIP_FLAG) -t "$(DESTDIR)/usr/bin/" "$(TOPDIR)/build/bin/dbp-cfg" 
 	install -m 755 $(STRIP_FLAG) -t "$(DESTDIR)/usr/bin/" "$(TOPDIR)/build/bin/dbp-cmd" 
